### PR TITLE
Added Uyghur language

### PR DESCRIPTION
**Pull request by boyaq**
Hi,

Thanks for the great library. Here is Uyghur translation of select2, hope you can merge it. Thanks a lot.

Best,

Alim

### CODE REVIEW

**Comment by arshaw** [comment]
note to self: figure out "eventLimitText" text

File: lang/ug-cn.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+$.fullCalendar.lang("ar", {
```
Comment (arshaw) [inline lang/ug-cn.js]: should the language code be `"ug-cn"` here?

**Comment by arshaw** [comment]
I would love to include this translation, but a prerequisite is that MomentJS must also support it. [Look a MomentJS's locale list](https://github.com/moment/moment/tree/develop/locale), it seems they do not support Uyghur yet. Am I looking for the wrong language code (is it `ug-cn`)? If MomentJS simply doesn't have it translated yet, could you contribute to their project?

**Comment by caseyjhol** [comment]
See https://github.com/moment/moment/pull/1834

**Comment by arshaw** [comment]
this can now be ported over to v4. i'll do that for the next release.

**Comment by arshaw** [comment]
this is released in v4.4.0:
https://github.com/fullcalendar/fullcalendar/releases/tag/v4.4.0

I renamed this to `ug`

HOWEVER, the browser must support the `ug` locale in order for this to work. On my computer Firefox does, but Chrome doesn't for example. If you want more consistent support, it's advised to use the moment plugin.

**Pull request merged**